common/src/main/java/org/cloudfoundry/identity/uaa/zone/IdentityZoneResolvingFilter.java

```diff
@@ -44,22 +43,28 @@ protected void doFilterInternal(HttpServletRequest request, HttpServletResponse
         IdentityZone identityZone = null;
         String hostname = request.getServerName();
         String subdomain = getSubdomain(hostname);
+        logger.debug(String.format("hostname=[%s], subdomain=[%s]", hostname, subdomain));
+        
         if (subdomain != null) {
             try {
                 identityZone = dao.retrieveBySubdomain(subdomain);
             } catch (EmptyResultDataAccessException ex) {
-                logger.debug("Cannot find identity zone for subdomain " + subdomain, ex);
+                logger.info(String.format("Cannot find identity zone for subdomain %s", subdomain));
```

Comment:
This should still be debug because the response body contains an error message. The identityZone will be null, and a message will be part of the HTTP response body (line 63)